src/pages/pass-finalisation/pass-finalisation.ts

```diff
@@ -95,14 +98,21 @@ export class PassFinalisationPage extends BasePageComponent {
       provisionalLicenseProvidedRadioChecked$: currentTest$.pipe(
         select(getPassCompletion),
         map(isProvisionalLicenseProvided),
+        tap((val) => {
+          if (val) this.form.controls['provisionalLicenseProvidedCtrl'].setValue('yes');
+        }),
```

Comment:
This sets the form control value when the page is navigated to. Feels a bit hacky perhaps?